### ISSUE TITLE
Using private repo after coursier

### ISSUE THREAD

**Issue by rmalarc** [almond-sh/almond#127]
Hi

I used to be able to use an external repository within jupyter notebook by using 

classpath.addRepository("http://nexus.myurl.com/")
classpath.add( "com.my.project" % "myproject" % "0.0.1")

What's the equivalent after coursier?

Thanks!

**Comment by jeremyplichta** [comment]
I am trying to figure this out as well. Any ideas?

**Comment by mallman** [comment]
Try

```scala
{
  import Resolvers._

  val customResolver =
    Resolver.Http(
      "my-custom-repo",
      "http://nexus.myurl.com/",
      MavenPattern,
      m2 = true)

  interp.resolvers() = defaultResolvers :+ customResolver
}
```

**Comment by lJoublanc** [comment]
This should probably be documented in the README.md. There is a link to the ammonite REPL, but the version shipped with jupyter-scala-0.4.2 is ammonite-0.8.3 (or ammonium), which doesn't support the `interp.repositories += syntax` which is documented there. This can be really confusing - took me over an hour til I found this PR!

**Comment by alexarchambault** [comment]
The current master, and releases under the "almond" name, work like [Ammonite](http://ammonite.io/#Ammonite-REPL), see [its documentation](http://ammonite.io/#Ammonite-REPL).
